[Maps] 7.8 documentation updates (#65502)
elastic/kibana: docs/maps/map-settings.asciidoc

[role="xpack"]
[[maps-settings]]
== Map settings

> To configure map settings, click *Map settings* in the application toolbar.

Reviewer: Although the intro text is ok as is, I think it might be better to add a little more information. 

Some suggestions:

Elastic Maps offers settings that let you configure how a map is displayed.  To access these settings, click *Map settings" in the application toolbar.

Looking to customize your map? Simply click *Map settings" in the application toolbar. You'll be able to configure zoom and spatial filters.

[float]
[[maps-settings-navigation]]
=== Navigation

*Zoom range*::
Constrain the map to the defined zoom range.

*Initial map location*::

> Configure the initial map center and zoom.

Reviewer: I'm wondering if you can move this sentence so it works as an intro sentence under the Navigation title.  So its similar to what you did in the Spatial filters section.

Author: Not sure this is a good intro sentence to navigation. There are some future configuration values that will require a broader introduction sentence.

* *Map location at save*: Use the map center and zoom from the map position at the time of the latest save.
* *Fixed location*: Lock the map center and zoom to fixed values.
* *Browser location*: Set the initial map center to the browser location.

[float]
[[maps-settings-spatial-filters]]
=== Spatial filters

Use spatial filter settings to configure how <<maps-spatial-filters, spatial filters>> are displayed.

image::maps/images/spatial_filters.png[]

*Show spatial filters on map*::

> Clear the checkbox to not show <<maps-spatial-filters, spatial filters>> on the map.

Reviewer: ```suggestion
Clear the checkbox so <<maps-spatial-filters, spatial filters>> do not appear on the map.
```

[role="xpack"]
[[maps-settings]]
== Map settings

Elastic Maps offers settings that let you configure how a map is displayed.

> To access these settings, click *Map settings" in the application toolbar.

Reviewer: ```suggestion
To access these settings, click *Map settings* in the application toolbar.
```
